test/performance/odp_timer_perf.c

```diff
@@ -442,9 +440,12 @@ static int set_timers(test_global_t *global)
 			timer_ctx_t *ctx = &global->timer_ctx[i][j];
 			odp_timer_start_t start_param;
 
+			nsec = START_NS + rand() % num_timer * period_ns;
```

Comment:
using random start times is not related to start/cancel time measurement, so it would be better to split it into separate patch. Usually, it would be better to use a fixed table of random numbers in a test application (instead of rand library call), as then the test pattern would be repeatable independent of OS, library or ODP version.

Reply:
I would like use a fixed table of random numbers for set_timers, as the set_timers function is not only used for start/cancel timer measurement, also used for profiling scan. I think there is no need to write another timer set function for sched mode.